Assets/_Standard Assets/Characters/Scripts/Physics/OpenCharacterController.cs

```diff
@@ -396,6 +396,12 @@ public void OnMoveLoop()
 		[SerializeField, Tooltip("Can character slide vertically when touching the ceiling? (For example, if ceiling is sloped.)")]
 		bool m_SlideAlongCeiling = true;
 
+		[SerializeField, Tooltip("Should the character keep a perfect velocity when sliding on walls?")]
+		bool m_OnWallsKeepVelocity = true;
```

Comment:
We prefer to keep features that most users are likely to want as opt-out rather than opt-in, so I suggest you invert this condition.

Try something like `m_SlowAgainstWalls`, and have it be true by default.

Reply:
I wouldn't have considered slow against walls being default. In fact going by forum, I find people get upset when their characters do not move at a consistent speed, so I advise having this setting disabled by default, which would also work best with people slotting this in as a replacement for the default character controller (which also doesn't slow against walls).